doc/manpages/bob-build-dev.rst

```diff
@@ -61,6 +61,9 @@ Options
 ``-b, --build-only``
     Don't checkout, just build and package
 
+``-^b, --no-build-only``
```

Comment:
There are many options without a counter part... if all of them can be set in the user-config they'll all need one... even the new `--no-logfile` option will need a `--enable-logfile`...

IMO a option to ignore (`-i`) the `config` section of the user-config's would give a greater usability? :thinking: